receiver/k8sclusterreceiver/internal/collection/nodes.go

```diff
@@ -83,6 +115,20 @@ var nodeConditionValues = map[corev1.ConditionStatus]int64{
 	corev1.ConditionUnknown: -1,
 }
 
+func nodeAllocatableValue(node *corev1.Node, allocatType corev1.ResourceName, logger *zap.Logger) (int64, bool) {
+	value, ok := node.Status.Allocatable[allocatType]
+	if !ok {
+		logger.Debug(string(allocatType) + " value not found in node")
+		return 0, false
```

Comment:
Let's `return 0, fmt.Errorf("Allocatable type %v not found in node %v", allocatableType, node.Name())`

The print debug from the outside function

Reply:
done